.github/workflows/stable-version-release.yml

```
@@ -0,0 +1,43 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    branches:
+      - 'v*'
+
+name: KubeArmor stable release 
```

Can you give this a different name?
```suggestion
name: KubeArmor stable release update
```